contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -590,6 +590,53 @@ pltsql_pre_parse_analyze(ParseState *pstate, RawStmt *parseTree)
 		}
 	}
 
+	if (parseTree->stmt->type == T_AlterTableStmt)
+	{
+		AlterTableStmt *atstmt = (AlterTableStmt *) parseTree->stmt;
+		ListCell *lc;
+		char *trig_schema;
+		const char *rel_schema;
+
+		foreach(lc, atstmt->cmds)
+		{
+			AlterTableCmd *cmd = (AlterTableCmd *)lfirst(lc);
+			if (cmd->subtype == AT_EnableTrig || cmd->subtype == AT_DisableTrig)
```

Comment:
no need to add any similar handling for ENABLE\DISABLE ALL here as,
`ENABLE/DISABLE TRIGGER ALL ON <table>` will enable/disable all the triggers present on that table
where as in case of `ENABLE/DISABLE TRIGGER <schema>.<trigger> ON <schema>.<table>`, we need to check whether both table and trigger are having same schema or not